crates/core/tedge_mapper/src/core/mapper.rs

```diff
@@ -33,6 +33,7 @@ pub async fn start_basic_actors(
         &mut mqtt_actor,
         &mqtt_schema,
         config.service.ty.clone(),
+        config.mqtt.topic_root.clone(),
```

Comment:
The same argument is given twice: first as an `MqttSchema`, second as `String`. Keep only the first one.

Reply:
fixed